### PR TITLE
add s3 external read permission to lambda function

### PR DESCRIPTION
**Pull request by jennyhliu**

### CODE REVIEW

**Comment by scisco** [comment]
@jennyhliu what's the use case for this permission?

**Comment by jennyhliu** [comment]
@scisco Ingest from a s3 provider.  When the lambda functions try to list the PDRs from a PDR location, or get the files from s3 location, the lambda functions need read permission to access these resources, or the operations will be denied. (I forgot the exact error message)

File: iam/cloudformation.template.yml [hunk 1]
```diff
@@ -213,6 +213,12 @@ Resources:
                   - !Sub "arn:aws:s3:::${ProtectedBucketName}/*"
                   - !Sub "arn:aws:s3:::${PublicBucketName}"
                   - !Sub "arn:aws:s3:::${PublicBucketName}/*"
+              # Allow read access to external s3 buckets
+              - Effect: Allow
+                Action:
+                  - s3:Get*
+                  - s3:List*
+                Resource: arn:aws:s3:::*
```
Comment (yjpa7145) [inline iam/cloudformation.template.yml]: Isn't this granting read access to _all_ buckets?  Is that something we want to allow?  @scisco, is there a better way to restrict this with our deployment method?

**Comment by scisco** [comment]
no longer relevant

**Pull request closed without merging**